lib/rest/preview/sync/service/syncList/syncListItem.js

```diff
@@ -740,7 +748,9 @@ SyncListItemContext.prototype.update = function update(opts, callback) {
   var deferred = Q.defer();
   var data = values.of({'Data': serialize.object(_.get(opts, 'data'))});
 
```

Comment:
nit: I opt for dropping the newline between `data` and `headers`.

Reply:
fixed